cmd/vl3-nse/vl3_connect.go

```diff
@@ -70,6 +70,117 @@ type vL3ConnectComposite struct {
 	nseControlAddr string
 }
 
+// newVL3ConnectComposite creates a new VL3 composite
```

Comment:
ah sure, I'll move it back. I think I misunderstood a review comment from @CosminNechifor . I changed the function to be a var so that I could mock it during unit testing. @CosminNechifor did you just want me to declare the var near the top of the file and leave the function implementation where it is?